apps/umpire/ChangeLog

```diff
@@ -0,0 +1,3 @@
+1.00: Beta version
+1.10: Added Heart Rate Monitoring
+1.20: General release
```

Comment:
We usually keep to `0.xx` for version numbers and I'd prefer that here as well.

If no one else has started using the app yet I suggest you restart at `0.01` and increment from there next time you update the app.